packages/foam-vscode/src/features/dataviz.ts

```diff
@@ -170,15 +171,15 @@ async function getWebviewContent(
   context: vscode.ExtensionContext,
   panel: vscode.WebviewPanel
 ) {
-  const datavizPath = [context.extensionPath, 'static', 'dataviz'];
+  const datavizPath = path.join(context.extensionPath, 'static', 'dataviz');
```

Comment:
the reason why i didn't do the join here is because I wanted to delegate to VS Code to find/use the right separator. Let's use VS Code capabilities where possible (this will also make it easier to make Foam a web extension)

Reply:
I agree with you rationale, but `path.join` *was used* in this case nevertheless, I just splitted its usage in two here. Indeed, this split wasn't deliberate, it's part of a previous refactoring that I'd done in order to replace `path.join` with `joinPaths` as you want. I reverted it because this is not the only  place where `path.join` is used to load foam resources, so I decided to keep the original code in places where foam was loading its resources, in order to avoid problems. If you prefer, I can replace all `path.join` usages with `uri.joinPath` or `path.joinPath` as I see fit (and as I'd originally done). I've commented about this in the PR description.

Reply:
oops I didn't notice, let's keep it as it was before for now (although I agree it needs to be changed), as the PR is already big and we should limit moving pieces here

Reply:
Sure.